WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -4,6 +4,7 @@ import Storage
 import Yosemite
 import Experiments
 import WooFoundation
+import Alamofire
```

Comment:
nit: can import the minimum like in `CardPresentPaymentsOnboardingUseCaseTests`
```suggestion
import enum Alamofire.AFError
```